components/team_card/team_card_storage.lua

```diff
@@ -0,0 +1,92 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:TeamCard/Storage
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Custom = require('Module:TeamCard/Custom')
+local String = require('Module:StringUtils')
+-- TODO: Once the Template calls are not needed (when RL has been moved to Module), deprecate Qualifier
+local Qualifier = require('Module:TeamCard/Qualifier')
+local Variables = require('Module:Variables')
+
+local TeamCardStorage = {}
+
+function TeamCardStorage.saveToLpdb(args, teamObject, players, playerPrize)
+	local team, teamTemplate
+	local image, imageDark
+
+	if type(teamObject) == 'table' then
+		if teamObject.team2 or teamObject.team3 then
+			team = 'TBD'
+		else
+			teamTemplate = teamObject.teamtemplate
+			team = teamObject.lpdb
+			image = args.image1
+			imageDark = args.imagedark1
+		end
+	end
+
+	local smwPrefix = args.smw_prefix or args['smw prefix'] or
+                      Variables.varDefault('smw prefix', Variables.varDefault('smw_prefix', ''))
+	local qualifierText, qualifierInternal, qualifierExternal = Qualifier.parseQualifier(args.qualifier)
+    local title = mw.title.getCurrentTitle().text
+    local endDate = Variables.varDefault('tournament_edate', Variables.varDefault('tournament_date'))
```

Comment:
Indenting

Reply:
Fixed, had swapped in the IDE to spaces by mistake.